sklearn/metrics/pairwise.py

```diff
@@ -127,11 +128,33 @@ def check_pairwise_arrays(
         - 'allow-nan': accepts only np.nan and pd.NA values in array. Values
           cannot be infinite.
 
-        .. versionadded:: 0.22
+        .. versionadded:: 0.20
```

Comment:
was this wrong?

Reply:
No, just a copy paste mistake on my end. I fixed that.